modules/metastorage/src/main/java/org/apache/ignite/internal/metastorage/impl/MetaStorageManagerImpl.java

```diff
@@ -338,7 +339,7 @@ private CompletableFuture<MetaStorageServiceImpl> initializeMetaStorage(Set<Stri
             var disruptorConfig = new RaftNodeDisruptorConfiguration("metastorage", 1);
 
             CompletableFuture<? extends RaftGroupService> localRaftServiceFuture = metaStorageNodes.contains(thisNodeName)
-                    ? startFollowerNode(metaStorageNodes, disruptorConfig)
+                    ? startVotingNode(metaStorageNodes, disruptorConfig)
```

Comment:
Technically, 'follower' doesn't seem correct as it might become a leader. I tried to make the terminology straight in the code I touched. The same should be probably done in other places, but in different PRs, IMO.

Reply:
I don't mean general "follower", but "followerNode" only. There are 2 variables to rename.

Reply:
There are methods called `beforeFollowerStop()` and `afterFollowerStop()`. The code that invokes them actually makes sure it passes a follower, not a leader, so they are named correctly, IMO. `followerNode` there is actually a follower, so nothing to rename.

But in `MetaStorageManagerImpl` we are starting a voting member, but we were calling it a follower (even though it could end up being a leader shortly after being started), so it was not correct to call it a 'follower'.

Reply:
Ok.